azure-bundle/LICENSE

```diff
@@ -483,25 +471,19 @@ License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: net.minidev  Name: accessors-smart  Version: 2.4.9
+Group: net.minidev  Name: accessors-smart  Version: 2.5.1
 Project URL: https://urielch.github.io/
 License: The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: net.minidev  Name: json-smart  Version: 2.4.10
+Group: net.minidev  Name: json-smart  Version: 2.5.1
 Project URL: https://urielch.github.io/
 License: The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: org.codehaus.woodstox  Name: stax2-api  Version: 4.2.1
-Project URL: http://github.com/FasterXML/stax2-api
-License: The BSD License - http://www.opensource.org/licenses/bsd-license.php
```

Comment:
Double checking this is no longer in the bundle

Reply:
Confirmed. We can remove ths.

Reply:
Thanks for checking 😄